dll/win32/shell32/wine/shell32_main.h

```diff
@@ -50,6 +50,16 @@ BOOL HCR_GetExecuteCommandW( HKEY hkeyClass, LPCWSTR szClass, LPCWSTR szVerb, LP
 BOOL HCR_GetIconW(LPCWSTR szClass, LPWSTR szDest, LPCWSTR szName, DWORD len, int* picon_idx);
 BOOL HCR_GetClassNameW(REFIID riid, LPWSTR szDest, DWORD len) DECLSPEC_HIDDEN;
 
+#ifdef __REACTOS__
+/* Current User */
+BOOL HCU_GetIconW(LPCWSTR szClass, LPWSTR szDest, LPCWSTR szName, DWORD len, int* picon_idx) DECLSPEC_HIDDEN;
+BOOL HCU_GetIconA(LPCSTR szClass, LPSTR szDest, LPCSTR szName, DWORD len, int* picon_idx) DECLSPEC_HIDDEN;
+
+/* Local Machine */
+BOOL HLM_GetIconW(int reg_idx, LPWSTR szDest, DWORD len, int* picon_idx) DECLSPEC_HIDDEN;
+BOOL HLM_GetIconA(int reg_idx, LPSTR szDest, DWORD len, int* picon_idx) DECLSPEC_HIDDEN;
```

Comment:
Why do we need ANSI versions of these ROS-specific internal functions?

Reply:
I just added them similarly to other similar functions originally came from Wine (`HCR_GetIconA`), to be more consistent. The fact we are not using them in the code indeed makes less sense to have them, but from other side, they potentially can be used as well in the shell32 code we'll write in the future.

Reply:
Wine's code doesn't actually make much sense for us. They have some ANSI helpers because they also want to cover Win9x OS compatibility with these. We don't need that.

Reply:
Ah, so this is the only reason... OK, I can remove them with no problem. :slightly_smiling_face:

Reply:
Done @HBelusca. :wink: